modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -538,7 +540,10 @@ public boolean isShowSubmitButton() {
 	}
 
 	public boolean isShowSuccessPage() throws PortalException {
-		if (SessionMessages.isEmpty(_renderRequest)) {
+		if (SessionErrors.contains(
+				_renderRequest, CaptchaTextException.class) ||
```

Comment:
@marcosapmf, can you please make it more generic using `!SessionErrors.isEmpty(...)`?

Thanks.

Reply:
Sure, @natocesarrego  :)